src/main/java/matveyeva/phonebook/crud/ContactCRUD.java

```diff
@@ -0,0 +1,135 @@
+package matveyeva.phonebook.crud;
+
+import matveyeva.phonebook.Validator;
+import matveyeva.phonebook.entity.Contact;
+import matveyeva.phonebook.entity.User;
+import matveyeva.phonebook.exception.InvalidContactException;
+import matveyeva.phonebook.exception.InvalidUserException;
+
+import java.io.*;
+import java.util.*;
```

Comment:
Generally, it's better to not use wildcard imports. There are a few reasons, code readability in reviews is one of them. Here's an example of another one http://www.javadude.com/posts/20040522-import-on-demand-is-evil/

Reply:
Additionally, you can setup autoformatter in idea. Save goodle code style format somewhere https://raw.githubusercontent.com/google/styleguide/gh-pages/intellij-java-google-style.xml and import it into idea in **settings>editor>code style>java>scheme (gear icon)> import intellij idea code style xml**. After that set tab size and indent size to 4, and in **Imports** tab `class count to use import with '*'`, same with names count.